src/main/java/com/twilio/oai/TwilioTerraformGenerator.java

```diff
@@ -23,6 +25,21 @@ public TwilioTerraformGenerator() {
         typeMapping.put("object", "string");
     }
 
+    @SuppressWarnings("unchecked")
+    @Override
+    public void processOpenAPI(OpenAPI openAPI) {
+        super.processOpenAPI(openAPI);
+        AtomicBoolean allCrudAvailable = new AtomicBoolean(false);
+        openAPI.getPaths().forEach((name, path) -> path.readOperations().forEach(operation -> {
```

Comment:
No need to iterate over `readOperations()` as it's not used. Recommend installing the SonarQube plugin for IntelliJ. It warns about such things.